backend/src/main/kotlin/no/nav/mulighetsrommet/api/kafka/KafkaFactory.kt

```diff
@@ -58,23 +73,18 @@ class KafkaFactory(private val db: DatabaseFactory) {
         }
     }
 
-    private fun getConsumerTopics() = kafkaConfig.config("topics").property("consumer").getList()
-
-    private fun configureConsumersTopics(topics: List<String>): List<KafkaConsumerClientBuilder.TopicConfig<String, String>> {
-        return topics.map { topic ->
+    private fun configureConsumersTopics(): List<KafkaConsumerClientBuilder.TopicConfig<String, String>> {
+        return topicMap.values.map { topic ->
+            // TODO: Må være en bedre måte å løse dependencies på her...
+            val eventProcessor = EventProcessor(topicMap, tiltakstypeService)
             KafkaConsumerClientBuilder.TopicConfig<String, String>()
                 .withLogging()
                 .withConsumerConfig(
                     topic,
                     stringDeserializer(),
                     stringDeserializer(),
-                    Consumer<ConsumerRecord<String, String>> { logTopicContent(it) }
+                    Consumer<ConsumerRecord<String, String>> { runBlocking { eventProcessor.process(it) } }
```

Comment:
Akkurat denne her "vet" jeg er feil måte å håndtere det på. Men jeg er litt usikker på hvordan `KafkaConsumerClientBuilder` fungerer her, dette må jeg undersøke. Det virker som den oppretter egene tråder for hver consumer client. Da er jeg litt usikker på hvordan det er med å opprette da coroutines inne i en tråd, om det i det hele tatt gir mening.